### PR TITLE
Update CDDA description on homepage

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
#### Summary

```SUMMARY: None```

#### Purpose of change

Several recent discussions on Reddit showed up that people often confuse CDDA with a roguelike, so we need to update description to avoid further confusion.

#### Describe the solution

Clean up game description on homepage.

#### Describe alternatives you've considered

Keep the confusion.

#### Additional context

#40944

### CODE REVIEW

**Comment by Hirmuolio** [comment]
CDDA is very much a roguelike.

Berlin interptation on what a roguelike is http://www.roguebasin.com/index.php?title=Berlin_Interpretation

**High value factors**

* Random environment generation
 The game world is randomly generated in a way that increases replayability. Appearance and placement of items is random. Appearance of monsters is fixed, their placement is random. Fixed content (plots or puzzles or vaults) removes randomness.

  Holds true for CDDA.

* Permadeath
You are not expected to win the game with your first character. You start over from the first level when you die. (It is possible to save games but the savefile is deleted upon loading.) The random environment makes this enjoyable rather than punishing.

  Holds true for CDDA.

* Turn-based
Each command corresponds to a single action/movement. The game is not sensitive to time, you can take your time to choose your action.

  Holds true for CDDA.

* Grid-based
The world is represented by a uniform grid of tiles. Monsters (and the player) take up one tile, regardless of size.

  Holds true for CDDA.

* Non-modal
Movement, battle and other actions take place in the same mode. Every action should be available at any point of the game. Violations to this are ADOM's overworld or Angband's and Crawl's shops.

  Holds true for CDDA.

* Complexity
The game has enough complexity to allow several solutions to common goals. This is obtained by providing enough item/monster and item/item interactions and is strongly connected to having just one mode.

  Holds true for CDDA through crafting and contruction.

* Resource management
You have to manage your limited resources (e.g. food, healing potions) and find uses for the resources you receive.

  Holds true for CDDA.

* Hack'n'slash
Even though there can be much more to the game, killing lots of monsters is a very important part of a roguelike. The game is player-vs-world: there are no monster/monster relations (like enmities, or diplomacy).

  Holds true for CDDA.

* Exploration and discovery
The game requires careful exploration of the dungeon levels and discovery of the usage of unidentified items. This has to be done anew every time the player starts a new game. 

  Mostly holds true for CDDA. Labs are dungeons of a sort and exploring cities is big part.

**Low value factors**

* Single player character
The player controls a single character. The game is player-centric, the world is viewed through that one character and that character's death is the end of the game.

  Holds true for CDDA.

* Monsters are similar to players
Rules that apply to the player apply to monsters as well. They have inventories, equipment, use items, cast spells etc.

  Holds true for CDDA. More for human enemies than monsters but still holds.
*
Tactical challenge
You have to learn about the tactics before you can make any significant progress. This process repeats itself, i.e. early game knowledge is not enough to beat the late game. (Due to random environments and permanent death, roguelikes are challenging to new players.) The game's focus is on providing tactical challenges (as opposed to strategically working on the big picture, or solving puzzles).

  Holds true for CDDA.

* ASCII display
The traditional display for roguelikes is to represent the tiled world by ASCII characters.

  Holds true for CDDA.

* Dungeons
Roguelikes contain dungeons, such as levels composed of rooms and corridors.

  Labs are dungeons of some sort.

* Numbers
The numbers used to describe the character (hit points, attributes etc.) are deliberately shown. 
 
  Holds true for CDDA.

**Comment by kevingranade** [comment]
* Random environment generation
True-ish, we have a LOT of hand-crafted content instead of procedural.
* Permadeath
Partially subverted, there is weakly-enforced permadeath, but there is also (optional) replay in the same world, and we do NOT reject more continuity-based options such as taking control of a NPC in the faction you control, which would entirely subvert permadeath *as defined by the Berlin Interpretation*.
* Turn-based
True
* Non-modal
Not true, and likely to be subverted further in the future.  There are a large number of modal and location-specific interfaces, and things like autotravel occurring at the overmap level are likely. Even shops are called out as being violations of this principle as a clarification, and we definitely have those.
* Resource management
True
* Hack'n'slash
Not true, there are explicitly monster-monster relations, it's a major game feature, Also killing monsters is explicitly not the only way to progress through the game. DDA considers monsters to be obstacles, not resources.
* Exploration and discovery
Partially true, Exploration is a major part of the game, but there is no meaningful per-game item discovery mechanic.

(low-value)
* Single player character
Not true, player controlled NPC factions and even more so NPC allies directly counters this.
* Monsters are similar to players
Not true, monsters cannot interact with items, have restricted movement modes, do not have access to the same style of attacks as the player (and vice versa).
* Tactical challenge
True
* ASCII display
Tiles are the dominant play mode based on download numbers. ASCII display remains present, but is niche.
* Dungeons
Sort of, the vast majority of play occurs on the surface.
* Numbers
Only some of them are shown, and in future we are more likely to trend toward fewer numbers intead of more.

Most importantly, if you bring up this definition as an argument for or against some feature, "Since DDA is a roguelike, you should...", that will be completely ignored, the Berlin Interpretation and the concept of "Roguelikeness" in general has nothing to do with development of the game.
That makes calling DDA a roguelike misleading.

**Comment by PlatoSoft** [comment]
Fair enough, I guess. The label doesn't really matter and if you disagree with the definitions so be it. I'll probably keep calling it a roguelike, no matter what the menus say. There are way less rogue-like games that are called roguelikes than CDDA. I was more shocked that you cared.

File: _config.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 title: "Cataclysm: Dark Days Ahead"
 email: kevin.granade@gmail.com
 description: > # this means to ignore newlines until "baseurl:"
-  Official homepage of Cataclysm: Dark Days Ahead open source roguelike game development project.
+  Official homepage of Cataclysm: Dark Days Ahead open source turn-based survival game development project.
```
Comment (kevingranade) [inline _config.yml:19]:
```suggestion
  Official homepage of Cataclysm: Dark Days Ahead open source turn-based survival RPG development project.
```

**Pull request merged**